libraries/botbuilder-dialogs-declarative/src/defaultLoader.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @module botbuilder-dialogs-declarative
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Configurable } from 'botbuilder-dialogs';
+import { CustomDeserializer } from './customDeserializer';
+import { ResourceExplorer } from './resources';
+
+export class DefaultLoader implements CustomDeserializer<Configurable, Record<string, unknown>> {
+    public constructor(private readonly _resourceExplorer: ResourceExplorer) {}
+
+    public load(config: Record<string, unknown>, type: { new (...args: unknown[]): Configurable }): Configurable {
+        const instance = new type();
+        Object.getOwnPropertyNames(config).forEach((k: string) => {
```

Comment:
I think `Object.entries(config).forEach(([key, value]) => 
  ...
{})` is more readable and clear.